csv-generator/Dockerfile

```diff
@@ -18,6 +18,10 @@ COPY ./pytest.ini ./
 COPY ./docker ./docker
 COPY ./csv_generator ./csv_generator
 
+# tests
```

Comment:
Not sure if we should have a separate `ci` build

Reply:
Becomes necessary when you have large images containing debug/testing tools, and that separates them into their own image descending from the one that gets published. For unit tests I wouldn't separate them yet